cache/cache.go

```diff
@@ -68,13 +69,13 @@ func RebuildCmd(c Cache, src, dst string) (err error) {
 	src = filepath.Clean(src)
 	src, err = filepath.Abs(src)
 	if err != nil {
-		return err
+		return fmt.Errorf("%s, absolute path failed for %s", err, src)
```

Comment:
@tboerger I'm sorry but I really don't have time right now to do these fixes for error messages, it will be great if you can fix those as needed, thanks.